src/sentry/api/endpoints/group_integration_details.py

```diff
@@ -69,18 +69,28 @@ def get(self, request: Request, group, integration_id) -> Response:
                 {"detail": "This feature is not supported for this integration."}, status=400
             )
 
+        installation = integration.get_installation(organization_id)
+        config = None
         try:
-            return Response(
-                serialize(
-                    integration,
-                    request.user,
-                    IntegrationIssueConfigSerializer(group, action, params=request.GET),
-                    organization_id=organization_id,
+            if action == "link":
+                config = installation.get_link_issue_config(group, params=request.GET)
+
+            if action == "create":
```

Comment:
Are there other actions? Should we raise if another action category is encountered?